cmd/utils/flags.go

```diff
@@ -1792,7 +1792,7 @@ func MakeGenesis(ctx *cli.Context) *core.Genesis {
 		genesis = core.DefaultRinkebyGenesisBlock()
 	case ctx.GlobalBool(GoerliFlag.Name):
 		genesis = core.DefaultGoerliGenesisBlock()
-	case ctx.GlobalBool(YoloV1Flag.Name):
+	case ctx.GlobalBool(YoloV2Flag.Name):
 		genesis = core.DefaultYoloV1GenesisBlock()
```

Comment:
Nit, pls rename to YoloV2. I know it didn't change, but seems nicer to get rid of all v1 mentions in the code.